### ISSUE TITLE
Vimium doesn't work on some pages

### ISSUE THREAD

**Issue by 1trapbox** [philc/vimium#4387]
## Vimium doesn't work on some pages
eg:
`brave://setting`
`https://chromewebstore.google.com`
`brave://newtab`
---
`Excluded URLs and keys` not set any

Is there a solution? 🙏

**Comment by remon2207** [comment]
It is not a bug, etc., because it is a specification that the extension cannot be used on the above page.

**Comment by ColorfullMadness** [comment]
When you are talking about specification I assume you mean some kind of global extensions specification that all extensions present in the official store have to follow, am I correct? 
If so, is there any way to circumvent this kind of limitations? I can understand why it is done but would like to know if there is a workaround since I trust this extension.

**Comment by gdh1995** [comment]
@n-kam you may modify Vimium's source code and add a feature to use the `browser.commands` API to register a browser-level shortcut to switch tabs. 

Here're some documents and examples:
- https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/commands
- https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/commands
- https://github.com/mdn/webextensions-examples/tree/main/commands

**Comment by philc** [comment]
The answer to this question is covered [here in the FAQ](https://github.com/philc/vimium/wiki/FAQ#why-doesnt-vimium-work-on-the-chrome-store-or-urls-starting-with-chrome).

I don't personally use Vimium's next/previous tab commands. Instead, I've configured system-wide shortcut which works across all apps with tabs (every browser, iTerm, Emacs). I've [outlined my workflow](https://github.com/philc/vimium/wiki/Tips-and-Tricks#have-nexttab-and-previoustab-command-work-on-all-pages-even-restricted-ones) on the "tips and tricks" page.

However, @gdh1995 makes an interesting suggestion to add a command. I've opened #4419 to discuss further.
